packages/ibm-products-web-components/src/components/user-avatar/user-avatar.scss

```diff
@@ -38,9 +38,38 @@ $block-class: #{$prefix}--user-avatar;
 
 //this mixin allows you to set background color based on the lighter themes such as g10 and white
 @mixin setLightBg($order, $color) {
-  :host .#{$block-class}--#{$order} {
+  :host([data-carbon-theme='g10']) .#{$block-class}--#{$order},
+  :host([data-carbon-theme='white']) .#{$block-class}--#{$order} {
+    @include setBgColor($color);
```

Comment:
I think `data-carbon-theme` comes specifically from our Storybook implementation so I don’t think we’d want to hardcode this value.

Might need to discuss our options a bit more here, but wondering if we are going to have to use a theme prop for this one after all.